wordpress/wp-content/plugins/cds-base/classes/Modules/FormRequestSite/src/handler.js

```diff
@@ -0,0 +1,45 @@
+(function ($) {
+    $("body").on("submit", "#request-form", function (e) {
+        e.preventDefault();
+        var form = $(this);
+        var url = form.attr('action');
+
+        // clear previous error messages
+        $(".gc-error-message").remove();
+
+        $.ajax({
+            type: "POST",
+            headers: {
+                'X-WP-Nonce': CDS_VARS.rest_nonce
+            },
+            url: url,
+            data: form.serialize(), // serializes the form's elements.
+            success: function (data) {
+                console.log(data);
+                if (data && data["error"]) {
+                    var errorEl = '<div id="request-error" class="gc-alert gc-alert--error gc-alert--validation" data-testid="alert" tabindex="0" role="alert">';
+                    errorEl += '<div class="gc-alert__body">';
+                    errorEl += '<h2 class="gc-h3">' + data.error_message + '</h2>';
+                    if(data['keys']) {
+                        errorEl += '<ol class="gc-ordered-list">';
+                        data['keys'].forEach(key => {
+                            errorEl += '<li><a href="#' + key + '" class="gc-error-link">Please complete the required field: ' + key +'</li>'
```

Comment:
We'll need to pass these strings from the sever so we can get them into the correct language.

Reply:
Okay, I'm just removing the second string here.

<img width="931" alt="Screen Shot 2022-03-08 at 09 56 08" src="https://user-images.githubusercontent.com/2454380/157263433-9c139ad5-b4b2-430a-ba10-d0dc1168e722.png">
<img width="928" alt="Screen Shot 2022-03-08 at 09 56 37" src="https://user-images.githubusercontent.com/2454380/157263436-604e22ab-ec6a-4a50-a02c-73f824315944.png">